### ISSUE TITLE
small issue in the preprocess glue data for finetuning

### ISSUE THREAD

**Issue by DikshaMeghwal** [pytorch/fairseq#1261]
A tiny bug in the preprocess_GLUE_tasks.sh script
--validpref "${DEVPREF//LANG/'label'}"  should be --validpref "${DEVPREF//LANG/label}" 
,i.e., there should be no quotes around the word 'label' in the filename for validation
https://github.com/pytorch/fairseq/blob/master/examples/roberta/preprocess_GLUE_tasks.sh#L176

**Comment by ngoyal2707** [comment]
Y=Thanks for reporting this, feel free to create a fix in PR, I will merge it

**Comment by DikshaMeghwal** [comment]
https://github.com/pytorch/fairseq/pull/1270
